blueprint/api.apib

```diff
@@ -481,6 +481,45 @@ Until the Code Corps platform is open to new organizations, only admin users can
 
     + Attributes (Unprocessable Entity Response)
 
+# Group Password
+
+This endpoint is a parent route for reset and forgot password endpoints
+
+
+## Reset [/reset]
+
+### Reset Password [POST]
+
++ Request
+
+    + Headers
+
+            Accept: application/vnd.api+json
+            Authorization: Bearer <token>
+
++ Response 201 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Reset Password Response)
```

Comment:
To conclude, leave the reset_password api docs out of this PR. The action needs some minor fixing, so we should add the docs separately as we fix that.